testing/endtoend/static-files/eth1/genesis.json

```diff
@@ -11,8 +11,8 @@
     "istanbulBlock": 0,
     "berlinBlock": 0,
     "londonBlock": 0,
-    "mergeForkBlock": 308,
-    "terminalTotalDifficulty": 616,
+    "mergeNetsplitBlock": 308,
+    "terminalTotalDifficulty": 0,
```

Comment:
TODO: remove this file. We can generate genesis.json now and shouldn't need it anymore.